### PR TITLE
Several improvements to "First 3D Game" tutorial

### PR DESCRIPTION
**Pull request by shane-smith**
The tutorial has been marked as compatible with the latest version - I just ran through it and verified that each step can be completed in Godot 4.1.3 stable.

A few fixes have been made, and other sections clarified to help first-time users of Godot.

I have made no changes to images - only text.

----

Note that this also fixes the "Animation > Copy" issue mentioned in https://github.com/godotengine/godot-docs/issues/6744

- The fix is located in Step 9, starting around Line 264
- The solution I've added to the documentation is the one suggested by @riidom here: https://github.com/godotengine/godot/issues/60848#issuecomment-1670398145
- If this PR is merged, it can be marked as done in the checklist on https://github.com/godotengine/godot-docs/issues/6744#issue-1574322830

### CODE REVIEW

File: getting_started/first_3d_game/01.game_setup.rst
Comment (Calinou) [inline getting_started/first_3d_game/01.game_setup.rst:31]: The project files should be updated for Godot 4, so this shouldn't be required.

You may see a notice because you're upgrading the project's *minor* version, but the dialog you see is different in this case:

image

File: getting_started/first_3d_game/03.player_movement_code.rst
Comment (Calinou) [inline getting_started/first_3d_game/03.player_movement_code.rst:13]:
```suggestion
it. In the popup, set the *Template* to *Empty* before pressing the *Create*
button. We set it to *Empty* because we want to write our own code for
player movement.
```

File: getting_started/first_3d_game/09.adding_animations.rst
Comment (Calinou) [inline getting_started/first_3d_game/09.adding_animations.rst:276]: The icon isn't "A+", it's more like "A>":

image

That said, I feel writing the icon as text isn't ideal as it can end up being confusing.

We should probably look into introducing inline icons in text in a future PR.

**Review by Calinou — COMMENTED** [review]
Should be good to merge after addressing suggestions.

File: getting_started/first_3d_game/01.game_setup.rst
Comment (shane-smith) [reply getting_started/first_3d_game/01.game_setup.rst:31]: Oh it gave me a warning about upgrading from a v3 project file. But I'm happy to revert this.

**Comment by shane-smith** [comment]
Thanks for the feedback! I'll work on this and get back to you.

File: getting_started/first_3d_game/09.adding_animations.rst
Comment (shane-smith) [reply getting_started/first_3d_game/09.adding_animations.rst:276]: Yes, I noticed this. It was written as A+ in other parts of the tutorials so I just copied that.

I can change this (and all the other occurrences that were already there) if that's the path forward.

**Comment by shane-smith** [comment]
I will squash this into a single commit (and rebase on the current master branch) and then it's ready for re-review.

**Comment by shane-smith** [comment]
It is ready for you now @Calinou - let me know if you have any further feedback 🙂

**Comment by skyace65** [comment]
This PR would supersede #8197. My preference is this one.

**Review by skyace65 — APPROVED** [review]
This looks good, I'd prefer to leave the "article outdated" warnings for now. There's still stuff on other pages that needs to be updated or fixed. but I'd be okay with merging this as is if others think it's fine.

I'll work on a PR to address the remaining issues once this one is merged.

**Comment by shane-smith** [comment]
I'll restore the warnings 👍

**Comment by docrinehart** [comment]
I don't think there will be any conflicts between this branch and the PR I submitted, but tagging this here for tracking purposes: 

#8772

**Comment by skyace65** [comment]
Salved by #8955

**Pull request closed without merging**
